renovate.json

```diff
@@ -6,7 +6,7 @@
   "separateMajorMinor": false,
   "prHourlyLimit": 2,
   "labels": [
-    "PR target: master-only",
+    "target: major",
```

Comment:
I find this confusing, because `master` can be for both a `major` or a `minor`.

Based on the [label docs](https://github.com/angular/angular/blob/master/docs/TRIAGE_AND_LABELS.md#pr-target) `target: major` should be used for breaking changes, while `target: minor` is for new features.

Since there is no 1 to 1 relation between a label and master branch. I feel the best fit is to use `minor`. Since that is how currently the workflow is on the angular/angular repo as far as I can tell. 

Example: https://github.com/angular/angular/pull/40058

Looks to me that `target: major` is more of a substitute for `PR target: master-only` and `flag: breaking change`.